pages/index.jsx

```diff
@@ -11,17 +15,19 @@ export default function HomePage({ games }) {
         <section>
           <StyledListLabel>In progress</StyledListLabel>
           <GameList progressLabel={"In Progress"} games={games} />
+          {!isSectionEmpty("In Progress") && <p>There are no games here...</p>}
```

Comment:
It's a lot of repetitive code here. Can you think of doing it inside the GameList component? You then only have to render three GameList components here. 
The best solution would be to have an array of progresses `["in progress", "planned","completed"]` and then map over this array and return a GameList for each entry. This would be perfect because you can expand this array later on and don't have to change any code if you e.g. make a new category like 'uninterresting' or something else.